README.md

```diff
@@ -1,111 +1,160 @@
+```
+pip install -e .
+```
 
-2.  Calculate PCA
+You can start the app while developing either by calling `cellxgene` or by calling `python -m server`.
 
-    sc.pp.pca(data) ## sc is scanpy.api
+## FAQ
 
-3.  Calculate nearest neighbors (depending on layout algorithm)
+> I'm following the developer instructions and get an error about "missing files and directories” when trying to build the client
 
-    ```
-    # For umap layout algorithm, you need to use the "umap" method for neighbors
-    sc.pp.neighbors(data, method="umap", metric="euclidean", use_rep="X_pca")
+This is likely because you do not have node and npm installed, we recommend using [nvm](https://github.com/creationix/nvm)
 
-    # For tsne layout algorithm, you can use either "umap" or "gauss"; we recommend "gauss"
-    sc.pp.neighbors(data, method="gauss", metric="euclidean", use_rep="X_pca")
-    ```
+## development roadmap
 
-4.  Save file
+`cellxgene` is still very much in development, and we've love to include the community as we plan new features to work on. We are thinking about working on the following features over the next 3-12 months. If you are interested in updates, want to give feedback, want to contribute, or have ideas about other features we should work on, please [contact us](#help-and-contact) 
 
-    ```
-    # cellxgene requires file to be named data.h5ad
-    data.write("data.h5ad")
-    ```
+- **Visualizaling spatial metadata** Image-based transcriptomics methods also generate large cell by gene matrices, alongside rich metadata about spatial location; we would like to render this information in `cellxgene`.
+- **Visualizing trajectories** Trajectory analyses infer progression along some ordering or pseudotime; we would like `cellxgene ` to render the results of these analyses when they have been performed
+- **Deploy to web**  Many projects release public data browser websites  alongside their publicatons; we would like to make it easy for anyone to deploy `cellxgene` to a custom URL with their own dataset that they own and operate.
+- **HCA Integration** The [Human Cell Atlas](https://humancellatlas.org) is generating a large corupus of single-cell expression data and will make it available through the Data Coordination Platform; we would like `cellxgene` to be one of several different portals for browsing these data
 
-## Contributing
+## contributing
 
-We warmly welcome contributions from the community. Please submit any bug reports and feature requests through github issues. Please submit any direct contributions via a branch + pull request.
+We warmly welcome contributions from the community! Please submit any bug reports and feature requests through [Github issues](https://github.com/chanzuckerberg/cellxgene/issues). Please submit any direct contributions by forking the repository, creating a branch (optional), and submitting a Pull Request.
```

Comment:
Stuff to consider adding:
* encourage PRs to contain test cases and documentation updates where applicable (caveat: the base project tests are still a work-in-progress)
* all dependencies and code contributions must conform to the base project licensing (MIT)

Reply:
good ideas, added both